### DISCUSSION TITLE
Bar counting or trading bars

### DISCUSSION THREAD

**Discussion by nqnga** [highfestiva/finplot#288] [Q&A] [answered]
Hi, 
Can I ask is there any supplement function for bar counting? or only count trading day?
Thanks

**Comment by highfestiva** [comment]
I don't understand the question. Do you mean Ctrl+Mouse Drag?

**Comment by nqnga** [comment, marked as answer]
There are two types of counting, counting the days based on calendar like from 1-jan to 10 jan it is 9 days. The other way of counting is only count the trading day, for e.g Apple do not trade on weekend, ad omit the new year day,  from 1-jan to 10-jan 2021, we only have 6 trading days. I want to know how to count 6 bars ( in daily time frame) or only trading days. and get the data, not from drawing on chart?
Thanks

**Reply by highfestiva** [reply]
You could use pandas for that. Since it's not from the chart, finplot is not required.

**Reply by nqnga** [reply]
I found the code :D not on the chart, but in pandas, Thanks!

**Reply by nqnga** [reply]
Can I ask how I can specific equal size of windows for chart:
ax,axv = fplt.create_plot('cur2', rows=2, maximize=False)
I want ax is same size as axv?
Thanks

**Reply by highfestiva** [reply]
https://github.com/highfestiva/finplot/issues/56

**Reply by nqnga** [reply]
Thank you very much!!!
